contributing.md

```diff
@@ -27,15 +27,15 @@ participants (see instructor guide for more details).
 >
 > - There is no need to download each file one by one.
 > - You can either download the repository as ZIP file (green button "Clone or download") if you do not plan to change files
-    and if you are sure that you don't need to browse the history of file changes.
+>   and if you are sure that you don't need to browse the history of file changes.
 > - **Possibly better: you clone the repository** (green button on the web or through GitHub Desktop or using command line) so that you
-    can apply and track changes and possibly share them later.
+>   can apply and track changes and possibly share them later.
```

Comment:
These are good points that are missed by the regex... oh well, I guess not worth fixing the regex.